lib/orwell/github/blog.ex

```diff
@@ -28,6 +28,19 @@ defmodule Orwell.GitHub.Blog do
   def posts_dir,
     do: Application.get_env(:orwell, :github_posts_dir)
 
+  @spec post_id(String.t) :: String.t
+  def post_id(path) do
+    case Regex.named_captures(~r|/(?<id>[0-9]+)[^/]*$|, path) do
```

Comment:
This is the second regex we use on this file. I'm starting to wonder if we should extract them to module variables.